scss/_mixins.scss

```diff
@@ -17,6 +17,9 @@
 	$border-width: map-get($button-dimensions, border-width);
 	$vertical-padding: 1/2 * (map-get($button-dimensions, height) - map-get($button-dimensions, font-size)) - $border-width;
 
+	background-size: map-get($button-dimensions, background-size);
+	height: map-get($button-dimensions, height);
+	min-width: map-get($button-dimensions, min-width);
 	padding: $vertical-padding map-get($button-dimensions, padding-right) $vertical-padding map-get($button-dimensions, padding-left);
 	font-size: map-get($button-dimensions, font-size);
 	line-height: map-get($button-dimensions, height) - $border-width * 2 - $vertical-padding * 2;
```

Comment:
If we refactor this again in future it might be worth doing something like this: http://sassbreak.com/stop-arguing-with-your-mixins/#comment-2560481986 using the spread operator and multiple parameters in the mixin?